app/views/courses/show.html.erb

```diff
@@ -60,7 +62,7 @@
 
       <%= render partial: 'courses/advice' %>
 
-      <%= render Courses::ApplyComponent.new(course) %>
+      <%= render Courses::ApplyComponent.new(course) if course.has_vacancies? || !CycleTimetable.mid_cycle? %>
```

Comment:
Does this mean that you'll have two elements with `id=section-apply` when `course.has_vacancies?` is false and `CycleTimetable.mid_cycle?` is false?

Reply:
The two elements should never render at the same time I think because of the additional logic in `apply_component.html.erb` so there should only ever be one element with `id=section-apply`